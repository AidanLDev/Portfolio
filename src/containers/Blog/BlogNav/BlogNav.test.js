import React from 'react';
import BlodNav from './BlogNav';
import { shallow } from 'enzyme';

it('Renders BlodNav Component Correctly', () => {
  const component = shallow(<BlodNav />);
  expect(component).toMatchSnapshot();
});
