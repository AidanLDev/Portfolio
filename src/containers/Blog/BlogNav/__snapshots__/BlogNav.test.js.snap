// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders BlodNav Component Correctly 1`] = `
<div
  className="container"
>
  <div
    className="rightSidebar"
  >
    <div
      className="searchBar"
    >
      <WithStyles(ForwardRef(TextField))
        label="Search Blogs"
        onChange={[Function]}
        style={
          Object {
            "margin": "auto",
          }
        }
        value=""
      />
    </div>
    <div
      className="weatherWidget"
    >
      <weatherWidget
        className="weatherWidget"
        weatherData={null}
      />
    </div>
  </div>
  <div
    className="mainContainer"
  >
    <h1>
      Blog Posts
    </h1>
    <blogCard
      description="This is just a placeholder"
      key="Blog posts will live here"
      route="pending"
      title="Blog posts will live here"
    />
  </div>
</div>
`;
